Plugins/Public/balancemagic/Main.cpp

```diff
@@ -455,23 +487,151 @@ void __stdcall HkCb_AddDmgEntry(DamageList *dmg, unsigned short p1, float damage
 	}
 }
 
+void __stdcall FireWeapon(unsigned int iClientID, struct XFireWeaponInfo const &wpn)
+{
+	returncode = DEFAULT_RETURNCODE;
+
+	if (!iLoadedBurstFires)
+		return;
+
+	IObjInspectImpl *obj = HkGetInspect(iClientID);
+	if (obj)
+	{
+		CShip* cship = (CShip*)HkGetEqObjFromObjRW((IObjRW*)obj);
+
+		for (short* slot = wpn.sArray1; slot != wpn.sArray2; slot++)
```

Comment:
can we get sArray1/sArray2 named?

Reply:
Probably could but that's an FLHook core thing, not a this-plugin thing